frontend/src/utils/index.js

```diff
@@ -0,0 +1,33 @@
+function filterbyPrice(products, sortBy) {
+  if (sortBy === "lth") {
+    return products.sort((firstitem, seconditem) => {
+      return firstitem.productprice - seconditem.productprice
+    })
+  }
+  if (sortBy === "htl") {
+    return products.sort((firstitem, seconditem) => {
```

Comment:
I would suggest to give proper naming like high-to-low or something similar. It would make your code more readable.